core/src/main/java/amino/run/oms/MicroServiceManager.java

```diff
@@ -215,6 +218,31 @@ public void setReplicaDispatcher(ReplicaID replicaId, EventHandler dispatcher)
         }
     }
 
+    /**
+     * Updates the microservice metrics for the given replica
+     *
+     * @param replicaId
+     * @param metrics
+     * @throws MicroServiceNotFoundException
+     * @throws MicroServiceReplicaNotFoundException
+     */
+    public void updateMetric(ReplicaID replicaId, Map<UUID, RPCMetric> metrics)
+            throws MicroServiceNotFoundException, MicroServiceReplicaNotFoundException {
+        getInstance(replicaId.getOID()).updateMetric(replicaId, metrics);
+    }
+
+    /**
+     * Gets the microservice metrics for the given replica
+     *
+     * @param replicaId
+     * @return
+     * @throws MicroServiceNotFoundException
+     */
+    public Map<InetSocketAddress, RPCMetric> getMicroServiceMetric(ReplicaID replicaId)
```

Comment:
nit: Need to remove 'MicroService' to make it consistent with the other methods updateMetric etc.